docs/proposals/version-documentation.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Documentation Versioning"
+linkTitle: "Documentation Versioning"
+weight: 1
+slug: documentation-versioning
+---
+
+- Author: [Jay Batra](https://github.com/jaybatra26)
+- Date: March 2020
+- Status: proposal
+
+## Problem
+
+In Cortex, documentation needs a lot of reordering. The idea is to have version documentation just like Prometheus.[`Prometheus`](https://prometheus.io/docs/introduction/overview/). Documentation is the main source of information for current contributors and first-timers. A properly versioned documentation will help everyone to have a proper place to look for answers before flagging it in the community. 
+
+In this proposal, we want to solve this. In particular, we want to:
+
+1. Version Documentation.
+2. Add or remove necessary and unnecessary data from it.
+3. Organize the documentation so that it's easily understandable.
+
+## Proposed solution
+
+The idea is to use a tool to organize current documentation. Prometheus uses netlify. 
+Rucio uses Sphinx. Other tools like Github Pages
```

Comment:
I think the following section is focusing on the wrong problem. We don't want to change how the website is built or deployed (unless strictly required, but let's say we should try to avoid it as much as possible), while we should focus the discussion on:

1. Which pages do we want to version?
2. For which versions we want to add a new versioning? Major, minor or patch version? I would say we want versioning for minor version (ie. `1.0`, `1.1`, ...)
3. How do we manage the versioning at website build time? How do we build the config for the version `1.0`, `1.1`, ... over the time?